project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -176,11 +183,41 @@ public void getRequest_placesAndUserLocationForwadedToScorer() throws Exception
     when(FETCHER.fetch(any(UserPreferences.class))).thenReturn(places);
     when(REQUEST.getParameter("location")).thenReturn("00.00000000,00.00000000");
 
-    servlet.doGet(REQUEST, RESPONSE);
+    servlet.doPost(REQUEST, RESPONSE);
 
     verify(SCORER).getScores(places, new LatLng(00, 00));
   }
 
+  @Test
+  // This test checks that storeUserPreferences is called with the expected parameters
+  // when the servlet gets a valid ID token
+  public void getRequest_validIdToken_userIdAndPreferencesForwardedForStoring() throws Exception {
```

Comment:
If it's now a post request, the function naming is inaccurate

Reply:
Totaly!